### PR TITLE
Add multiplex tests to t0130.

### PR DESCRIPTION
**Pull request by Kubuxu**
For some reasons multiplex won't transfer those files, it hangs.

--- 
CHANGELOG:
test go-multiplex stream muxer in multinode sharness test.

### CODE REVIEW

**Comment by lgierth** [comment]
I figure this is more of a bug report than to-be-merged eh? The tests LGTM though :)

**Comment by Kubuxu** [comment]
@whyrusleeping asked me to write this test and it turns out it is failing.

**Comment by whyrusleeping** [comment]
CHANGELOG:
Test go-multiplex stream muxer in multinode sharness test.

**Comment by Kubuxu** [comment]
I added it to initial PR comment, do you think it is better idea.

File: test/sharness/t0130-multinode.sh [hunk 1]
```diff
@@ -87,6 +87,13 @@ test_expect_success "set up tcp testbed" '
 	iptb init -n 5 -p 0 -f --bootstrap=none
 '
 
+# test multiplex muxer
+export LIBP2P_MUX_PREFS="/mplex/6.7.0 /yamux/1.0.0 /spdy/3.1.0"
```
Comment (whyrusleeping) [inline test/sharness/t0130-multinode.sh]: just to be a bit more certain of what we're testing, lets remove yamux and spdy from this list.

Comment (Kubuxu) [inline test/sharness/t0130-multinode.sh]: k

**Comment by whyrusleeping** [comment]
merged in #3582

**Pull request closed without merging**